deployment_report/templates/viya_deployment_report.html.j2

```diff
@@ -299,14 +308,22 @@
   <h2>Configurations</h2>
   {# Configurations: Content #}
 
-  {# Cluster Overview: Configmaps Accordion #}
-  {% set kind_name = "Configmaps" %}
   <div class="jq-accordion">
-    <h2>Configmaps</h2>
-    <div>
-      {% for name, info in report_data.kubernetes.configMaps.items() %}
+  {# Configurations: Configmaps Accordion #}
+  <h2>Configmaps</h2>
```

Comment:
Let's change `<h2>Configmaps</h2>` to `<h2>ConfigMaps</h2>` to better align with the way k8s formats the name.